building/device-installation.rst

```diff
@@ -0,0 +1,34 @@
+
+Device Installation
+===================
+
+In this section we will flash and install our builded images combined with a
+chosen rootfs. For this you need a ``hybris-boot.img``, a ``system.img``, a
+capable android device and a downloaded rootfs.
+
+Flashing the Halium system on the device
+----------------------------------------
+
+.. Note::
+
+   ``BUILDDIR`` is the location of your Halium tree checkout
+
+First you need the halium install tool::
+
+   wget https://raw.githubusercontent.com/Halium/halium-scripts/master/halium-install
```

Comment:
why not add halium-scripts here: https://github.com/Halium/android

Reply:
Well if you create a pull request to add it to the main manifest, I can change it here of course :)